travel_platform/izitravel_t.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+import ConfigParser
+import os
+import requests
+import unittest
+
+
+from izi_travel_client import IziTravelApiClient
+from izi_travel_client import IziTravelApiError
+
+
+class IziTravelTest(unittest.TestCase):
+    def setUp(self):
+        config = ConfigParser.ConfigParser()
+        config.read("{}/{}".format(os.getcwd(), 'configurations.cfg'))
```

Comment:
well, it's just a string? If config variable is just a string - that's an issue, but I guess the problem is that you don't store result in some variable, that's why you can't get appropriate key from your `config`.